contribute/developer-guide.md

```diff
@@ -112,6 +112,10 @@ To follow the tests in the browser while they're running, add the `BROWSER` and
 BROWSER=1 SLOWMO=1 yarn e2e-tests
 ```
 
+### Building on Windows
```

Comment:
Is this the right place to put this section? Is it even still correct?

Reply:
This is the correct page to move it to :+1: But not sure of this, currently in `## Test Grafana` section. How about moving it to `#### Building on Windows` in the `### Backend` section above?

Reply:
Not sure if this is the right place. People will Google or search for this so as long as they can find it, it's fine.We moved it from here (when it was on the docs site and not in the GitHub repo) to the Windows docs a while ago. When you Google it, you can see that the results show both pages (and a really old version of the page):

![image](https://user-images.githubusercontent.com/434655/70158238-33aca480-16b7-11ea-93e3-7de3e7dd8ddf.png)

Reply:
Moved.

Reply:
Moved it to Backend.